util/src/main/java/com/google/cloud/hadoop/util/ApiErrorExtractor.java

```diff
@@ -205,6 +207,18 @@ public boolean rateLimited(IOException e) {
     return false;
   }
 
+  /**
+   * Determines if a given Throwable is caused by Quota Exceeded. Recursively checks
+   * getCause() if outer exception isn't an instance of the correct class.
+   */
+  public boolean quotaExceeded(IOException e) {
+    ErrorInfo errorInfo = getErrorInfo(e);
+    if(errorInfo !=null){
```

Comment:
Please, format this change with google java formatter: https://github.com/google/google-java-format

Reply:
Reformatted with the IntelliJ google-java-format plugin

Reply:
Please format only changes lines, not whole file: https://github.com/google/google-java-format/blob/master/scripts/google-java-format-diff.py

Reply:
Sorry for that, I have created a new PR for the change, will close this one once that get passed.